pythonFiles/tests/debug_adapter/test_ptvsd_folder_name_functional.py

```diff
@@ -0,0 +1,60 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import sys
+
+if sys.version_info[:2] != (3, 7):
+    import unittest
+
+    raise unittest.SkipTest("PTVSD wheels shipped for Python 3.7 only")
+
+import os.path
+import pytest
+import subprocess
+
+from .. import PROJECT_ROOT, SRC_ROOT
+from packaging.requirements import Requirement
+
+PYTHONFILES = os.path.join(SRC_ROOT, "lib", "python")
+REQUIREMENTS = os.path.join(PROJECT_ROOT, "requirements.txt")
+ARGV = ["python", os.path.join(SRC_ROOT, "ptvsd_folder_name.py")]
+PREFIX = "ptvsd=="
+
+with open(REQUIREMENTS, "r", encoding="utf-8") as reqsfile:
+    for line in reqsfile:
+        if line.startswith(PREFIX):
+            VERSION = line[len(PREFIX) :].strip()
```

Comment:
only need the first one?

```suggestion
            VERSION = line[len(PREFIX) :].strip()
            break
```